server/doc.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+mongoose.promise = Promise;
+const Schema = mongoose.Schema;
+const date = require('date-and-time');
+
+const docSchema = new Schema({
+  name: {type: String, required = true},
+  file: {type: String, required = true},
+  owner_id: {type: Number, required = true},
+  locked: Boolean,
+  revision_id: Number,
```

Comment:
This should be a list of foreign keys i.e 
revisions:[
      {type: Schema.Types.ObjectId, ref: 'Revisions'}
]

Reply:
changed